test/parallel/test-file-read-noexist.js

```diff
@@ -20,13 +20,19 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-const common = require('../common');
-const fixtures = require('../common/fixtures');
+require('../common');
 const assert = require('assert');
+const fixtures = require('../common/fixtures');
 const fs = require('fs');
+const { describe, it } = require('node:test');
 
-const filename = fixtures.path('does_not_exist.txt');
-fs.readFile(filename, 'latin1', common.mustCall(function(err, content) {
-  assert.ok(err);
-  assert.strictEqual(err.code, 'ENOENT');
-}));
+describe('fs.readfile', () => {
+  it('should return ENOENT error code for non existing file', (_, done) => {
+    const filename = fixtures.path('does_not_exist.txt');
+    fs.readFile(filename, 'latin1', function(err, _) {
```

Comment:
I'd recommend keeping the `common.mustCall(...)` here or replacing it with a `mock.fn` and check that the function was called.

Reply:
Yes, it makes sense, we re-add it, thanks.
- [ ] Re-add `common.mustcall(...)` function.